cmd/celestia-appd/cmd/root.go

```diff
@@ -77,7 +78,13 @@ func NewRootCmd() *cobra.Command {
 				return err
 			}
 
+			// change tendermint config for celestia app
+			// set broadcast timeout to be 50 seconds in order to avoid timeouts for long block times
+			// set max body bytes for transaction to be 8 Mb
```

Comment:
Note this comment is still inaccurate ` 8 Mb` => `8 MiB`. On re-review, the comment doesn't provide any information that isn't evident in the of code so proposal to remove it entirely

```
			tmCfg.RPC.MaxBodyBytes = int64(8388608) // 8 MiB
```